src/docs/components/styling.md

```diff
@@ -45,7 +45,27 @@ export class ShadowComponent {
 
 ### Things to remember with Shadow DOM
 
-- QuerySelector: When using Shadow DOM and you want to query an element inside your web component you must use `this.el.shadowRoot.querySelector()`. This is because all of your DOM inside your web component is in a shadowRoot that Shadow DOM creates.
+- QuerySelector: When using Shadow DOM and you want to query an element inside your web component, you must first use the [`@Element` decorator](https://stenciljs.com/docs/host-element#element-decorator) to gain access to the host element, and then you can use the `shadowRoot` property to perform the query. This is because all of your DOM inside your web component is in a shadowRoot that Shadow DOM creates. For example:
```

Comment:
```suggestion
- QuerySelector: When using Shadow DOM and you want to query an element inside your web component, you must first use the [`@Element` decorator](/docs/host-element#element-decorator) to gain access to the host element, and then you can use the `shadowRoot` property to perform the query. This is because all of your DOM inside your web component is in a shadowRoot that Shadow DOM creates. For example:
```